VueJS/components/List.js

```diff
@@ -1 +1,14 @@
-export default Vue.component('list', {})
+export default Vue.component('list', {
+    template: `<div>
+		        <ul>
+			        <list-item
+			    	v-for="(item, index) in items"
+			    	v-bind:item="item"
+			    	v-on:save="$emit('save', {listIndex: index, message: $event})"
```

Comment:
I would prefer to handle the `save` event in this very component. The `List` should have the responsibility to re-render the `ListItem` component. If you alter the list of data objects in this component, you don't have to bubble up `$event`.